src/Umbraco.Web/PublishedCache/NuCache/DataSource/LazyCompressedString.cs

```diff
@@ -0,0 +1,31 @@
+﻿using K4os.Compression.LZ4;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Umbraco.Web.PublishedCache.NuCache.DataSource
+{
+    public class LazyCompressedString
```

Comment:
keep things internal for now that we don't need to make public

Reply:
changed to internal